releng/plugins/org.polarsys.capella.settings/oomph/Capella.setup

```diff
@@ -334,13 +338,17 @@
           <repository
               url="http://download.eclipse.org/egf/updates/1.6.1/2019-06"/>
           <repository
-              url="https://download.eclipse.org/sirius/updates/stable/6.3.3-S20200806-045005/2019-06/"/>
+              url="https://download.eclipse.org/sirius/updates/releases/6.3.3/2019-06/"/>
```

Comment:
We seem to use https://download.eclipse.org/sirius/updates/stable/6.4.0-S20201119-043506/2020-06 in the current TP